src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsSelector.js

```diff
@@ -282,15 +303,15 @@ function MoneyRequestParticipantsSelector({
                 ref={forwardedRef}
                 headerMessage={headerMessage}
                 boldStyle
-                shouldShowConfirmButton={shouldShowConfirmButton && isAllowedToSplit}
-                confirmButtonText={translate('iou.addToSplit')}
+                shouldShowConfirmButton={isAllowedToSplit}
                 onConfirmSelection={navigateToSplit}
                 textInputLabel={translate('optionsSelector.nameEmailOrPhoneNumber')}
                 textInputAlert={isOffline ? `${translate('common.youAppearToBeOffline')} ${translate('search.resultsAreLimited')}` : ''}
                 safeAreaPaddingBottomStyle={safeAreaPaddingBottomStyle}
                 shouldShowOptions={isOptionsDataReady}
                 shouldPreventDefaultFocusOnSelectRow={!Browser.isMobile()}
                 shouldDelayFocus
+                footerContent={isAllowedToSplit && footerContent}
```

Comment:
Won't this cause console error?
```suggestion
                footerContent={isAllowedToSplit ? footerContent : undefined}
```